maro/cli/grass/utils/copy.py

```diff
@@ -16,28 +16,36 @@
 logger = CliLogger(name=__name__)
 
 
-def copy_files_to_node(local_path: str, remote_dir: str, admin_username: str, node_ip_address: str) -> None:
+def copy_files_to_node(
+    local_path: str, remote_dir: str,
```

Comment:
one line one parameter

Reply:
like this? Looks like a waste of space
![Screen Shot 2020-12-21 at 21 18 44](https://user-images.githubusercontent.com/18544909/102781150-1df94280-43d2-11eb-8a2b-b9533f8ac358.png)